lib/cocina_display/cocina_record.rb

```diff
@@ -78,6 +78,11 @@ def related_resources
       @related_resources ||= path("$.description.relatedResource[*]").map { |res| RelatedResource.new(res) }
     end
 
+    def contact_information
```

Comment:
Is this method being used anywhere yet? Can I change this to `access_contact_display_data` in my implementation to conform with method naming conventions for similar methods?